src/api/spec/models/bs_request_spec.rb

```diff
@@ -68,4 +68,30 @@
       end
     end
   end
+
+  describe '#changestate' do
+    let(:request) { create(:bs_request) }
+    let(:admin) { create(:admin_user) }
+
+    context 'for wiping request' do
+      before do
+        request
+
+        # assign the admin after the first call of 'request'. this is important to avoid errors
+        User.current = admin
```

Comment:
It doesn't have to be an `admin`. It also can be the guy that opened the review but I didn't try things out and was just using `admin` since I knew it would always work. I think it doesn't matter what user I use here. I guess `admin` is fine.